### PR TITLE
docs: 8.2 upgrade for Tasklist

### PR DESCRIPTION
**Pull request by marcosgvieira**
## What is the purpose of the change

For the new feature "Start Process from tasklist" to work on Tasklist new configuration is needed.

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by marcosgvieira** [comment]
@dlavrenuek can you please have a look on the new environment variables for Tasklist, I would like to keep the same wording as Identity is using.

File: docs/guides/update-guide/810-to-820.md
Comment (dlavrenuek) [inline docs/guides/update-guide/810-to-820.md:35]: Are these the names that are used directly by Tasklist or in the help charts? For the flag I would omit Identity, so `CAMUNDA_TASKLIST_RESOURCE_PERMISSIONS_ENABLED` instead of `CAMUNDA_TASKLIST_IDENTITY_RESOURCE_PERMISSIONS_ENABLED` since resource permission or resource authorizations is a new concept globally in C8 and not just related to Identity, but it's not a "hard" opinion from my side.

**Comment by christinaausley** [comment]
I made some adjustments in my latest commit in a few places where I was confused -- could you review to ensure I have clarified this properly?

File: docs/guides/update-guide/810-to-820.md
Comment (dlavrenuek) [inline docs/guides/update-guide/810-to-820.md]: We will document how to enable the feature in the concepts section, so it is easier discovered by users.

@christinaausley a PR with the new page is on the way.

```suggestion
    - To enable this variable in Tasklist, Resource authorizations must be [enabled in Identity](../../concepts/access-control/resource-authorizations/).
```

**Review by dlavrenuek — CHANGES_REQUESTED** [review]

File: docs/guides/update-guide/810-to-820.md
Comment (marcosgvieira) [reply docs/guides/update-guide/810-to-820.md]: Those are env vars. On the name, I added the IDENTITY as all other IDENTITY configurations are working like that in tasklist.

File: docs/guides/update-guide/810-to-820.md
Comment (marcosgvieira) [reply docs/guides/update-guide/810-to-820.md]: @christinaausley and @dlavrenuek should we hold this PR until the new page is created or should we merge it?

Comment (christinaausley) [reply docs/guides/update-guide/810-to-820.md]: @dlavrenuek @marcosgvieira Up to you -- if you want to go ahead and get this PR out the door we can always start fresh with a new one.

**Review by christinaausley — CHANGES_REQUESTED** [review]
To get this working we'll just need to complete the following:

1. Resolve merge conflicts.
2. Adjust resource authorization links, as these are now broken.

Let me know if I can assist -- happy to help!

File: docs/guides/update-guide/810-to-820.md
Comment (dlavrenuek) [reply docs/guides/update-guide/810-to-820.md]: @marcosgvieira we will have cross-references f.e. from [this PR](https://github.com/camunda/camunda-platform-docs/pull/1897), so you can go ahead and merge if the only problem is the missing page

**Comment by akeller** [comment]
🥚  Merging -> https://camunda.slack.com/archives/C026U8GBNSW/p1680812269564129

**Pull request merged**
